tpu-client-next/tests/connection_workers_scheduler_test.rs

```diff
@@ -46,9 +48,10 @@ fn test_config(validator_identity: Option<Keypair>) -> ConnectionWorkersSchedule
         stake_identity: validator_identity,
         num_connections: 1,
         skip_check_transaction_age: false,
-        worker_channel_size: 2,
+        worker_channel_size: 100,
```

Comment:
Due to the change from `send_transactions` to `try_send_transactions`, had to increase it to avoid surprises in CI.

Reply:
Maybe add a comment here in the code?
You have collected some knowledge about this configuration parameter.
But, unless you write it down in a comment, nobody else will be able to benefit from it.
No one will find this PR comment after the PR is closed.